geoengine/types.py

```diff
@@ -573,6 +595,11 @@ def __repr__(self) -> str:
         '''Display representation of a unitless measurement'''
         return str(self)
 
+    def to_dict(self) -> Dict[str, Any]:
```

Comment:
and here

Reply:
04e3b2b514c10eeaa30770a65b6b2d4663500498